install/kubernetes/helm/istio/charts/certmanager/templates/deployment.yaml

```diff
@@ -18,6 +18,8 @@ spec:
 {{ toYaml .Values.podLabels | indent 8 }}
 {{- end }}
       annotations:
+        sidecar.istio.io/inject: "false"
+        scheduler.alpha.kubernetes.io/critical-pod: ""
```

Comment:
I wouldn't call it a critical pod.

AFAIK istio-system is not injected by default - which reminds me that we haven't tested certmanager when mtls is globally enabled ( likely to fail unless we add a destination policy - which may be impossible since we don't know the
name of the dynamic created ingress)

Reply:
@costinm here I want to take it as critical pod, because I do not want this pod to be killed when the pod is running, so that we can make sure this service is always available.

I did not found the logic that istio ignore `istio-system` when inject, but if we do not set `sidecar.istio.io/inject`, the default value will be `true`.

For cert manager, I think it was not tested fully for istio, and it was currently disabled by default, I think we can do more test in post 1.0